hooks/check_skip_env.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+import re
+import sys
+import argparse
+import logging
+
+
+logging.basicConfig(format='%(asctime)s [%(levelname)s] %(message)s', level=logging.INFO)
+
+
+def has_setenv_skip(fpath):
+    with open(fpath) as f:
+        for line in f:
+            if re.match(r'^\s+os.Setenv\(\"SKIP_', line):
```

Comment:
It's not likely, but perhaps possible someone calls `os.Setenv` right at the start of a line of code? Also, there may be extra whitespace.

```suggestion
            if re.match(r'^\s*os.Setenv\s*\(\s*\"SKIP_', line):
```

Also, consider checking for `os.Setenv("TERRATEST_REGION`

Reply:
> It's not likely, but perhaps possible someone calls os.Setenv right at the start of a line of code? Also, there may be extra whitespace.

This would be formatted away with `go fmt`, so I don't think it is necessary.

> Also, consider checking for os.Setenv("TERRATEST_REGION

Added in! e20ca37